daal4py/sklearn/cluster/_k_means_0_23.py

```diff
@@ -184,6 +192,15 @@ def _daal4py_k_means_fit(X, nClusters, numIterations, tol, cluster_centers_0, n_
 
     flag_compute = 'computeAssignments|computeExactObjectiveFunction'
     best_labels, best_inertia = _daal4py_k_means_predict(X, nClusters, best_cluster_centers, flag_compute)
+
+    distinct_clusters = len(set(best_labels))
```

Comment:
Due to perf. considerations, try to avoid usage of default python function if corresponding numpy's exist. In this case, it's ndarray.n_unique